staff/lgimeneza/css-layout-simple/styles/main.css

```diff
@@ -0,0 +1,36 @@
+body{
+    background-color: blue;
+    height: 100%;
+    width: 100%;
+    margin: 0px;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+html{
+    height: 100%;
+    width: 100%;
+}
+
+.myHeader{
+    background-color: crimson;
+    text-align: center;
+    height: 15%;
+}
+.myFooter{
+    background-color: cadetblue;
+    text-align: center;
+    height: 15%;
+}
+.myAside{
+    background-color: bisque;
+    float: left;
+    width: 30%;
+    height: 100%;
+}
+.mainSection{
+    background-color: azure; 
+    height: 100%;
+}
```

Comment:
Should you use the selector by class when there is not more than one element using that class? investigate the selector by tag